node/src/main/scala/com/wavesplatform/api/common/CommonAccountsApi.scala

```diff
@@ -188,8 +188,8 @@ object CommonAccountsApi {
         leaseId,
         ld.sourceId,
         ld.sender.toAddress,
-        blockchain.resolveAlias(ld.recipient).orElse(resolveDisabledAlias(leaseId)).explicitGet(),
-        ld.amount,
+        blockchain.resolveAlias(ld.recipientAddress).orElse(resolveDisabledAlias(leaseId)).explicitGet(),
```

Comment:
```suggestion
        ld.recipientAddress,
```
`ld.recipientAddress` is `Address`, so `blockchain.resolveAlias(ld.recipientAddress)` returns `Right` and we won't fall into `resolveDisabledAlias(leaseId)`